tests/fixtures/name-matches-to-wikidata.json

```diff
@@ -0,0 +1,96 @@
+{
+  "compareFunction": "name-matches-to-wikidata",
+  "fixtures": [
+    {
+      "description": "Test OSM name different from Wikidata name",
+      "expectedResult": {
+        "result:name_matches_to_wikidata": false
+      },
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "osm:id": 427818536,
+          "osm:type": "way",
+          "name": "Central Park is now something else",
+          "wikidata": "Q160409",
+          "osm:version": 1
+        },
+        "geometry": null
+      },
+      "oldVersion": null
+    },
+    {
+      "description": "Test OSM name matches with aliases on Wikidata",
+      "expectedResult": {},
```

Comment:
This design essentially does not differentiate between no data and a positive result. 

If the goal is to have every changeset reviewed by the community, comparators should pass on as much useful knowledge as possible to a human reviewer to make the final decision. This comparator has essentially done the tedious effort of looking up Wikidata and comparing names, witholding this finding will lead to duplicate human effort on the same activity. What do we gain by this?